frontend/src/metabase/collections/components/CollectionMenu/CollectionMenu.tsx

```diff
@@ -23,13 +25,27 @@ export const CollectionMenu = ({
   isPersonalCollectionChild,
   onUpdateCollection,
 }: CollectionMenuProps): JSX.Element | null => {
+  // we don't want any of the items, we just want to know how many there are in the collection
+  const query = useListCollectionItemsQuery({ id: collection.id, limit: 0 });
```

Comment:
nit: for a bit more clarity you could put the comment on `limit: 0` so it's super clear what you're describing. But it's also clear enough to anyone who thinks about it for 5 seconds